### PR TITLE
[fix] Handle negative big ints correctly

### PR DESCRIPTION
**Pull request by cedric-cordenier**

### CODE REVIEW

File: pkg/values/pb/values.proto [hunk 1]
```diff
@@ -13,10 +13,15 @@ message Value {
     List list_value = 5;
     string decimal_value = 6;
     int64 int64_value = 7;
-    bytes bigint_value = 8;
+    BigInt bigint_value = 8;
```
Comment (bolekk) [inline pkg/values/pb/values.proto:16]: can we use 9 and mark 8 as reserved?

File: pkg/values/pb/values.proto [hunk 2]
```diff
@@ -13,10 +13,15 @@ message Value {
...
   }
 }
 
+message BigInt {
+     bytes abs_val = 1;
+     int64 sign = 2;
+}
```
[the remark this answers is not in the document]
Comment (cedric-cordenier) [reply pkg/values/pb/values.proto:23]: It's totally isolated; I'm not sure the deduplication here is worth it

File: pkg/values/pb/values.proto [hunk 3]
```diff
@@ -13,10 +14,15 @@ message Value {
     List list_value = 5;
     string decimal_value = 6;
     int64 int64_value = 7;
-    bytes bigint_value = 8;
```
Comment (jmank88) [inline pkg/values/pb/values.proto:16]: I think there is a formal way to reserve or deprecate this field

**Review by krehermann — APPROVED** [review]

**Review by bolekk — APPROVED** [review]

File: pkg/values/pb/values.proto (hunk 3 above)
Comment (jmank88) [reply pkg/values/pb/values.proto:16]: Oh I just expected it to be inline here inside of the `oneof` - does it matter?

**Pull request merged**